.github/workflows/ci.yml

```diff
@@ -66,31 +64,9 @@ jobs:
       - name: Download Go modules
         run: go mod download
 
-      - name: Install development tools
-        run: make init
-
-      - name: Generate files
-        run: make gen gen-alertmanager
-
-      - name: Check differences
-        run: |
-          git status
-          git diff --exit-code
-
-      - name: Update API compatibility descriptors.
-        run: |
-          # log if descriptors changed, useful for "update descriptors" PRs
-          make descriptors
-          git diff --text
-
       - name: Check build
         run: make release
 
-      - name: Run linters
-        env:
-          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: make ci-reviewdog
```

Comment:
@BupycHuk it seems that this part was missed

Reply:
oh, shit